src/streaming-client.ts

```diff
@@ -53,14 +56,20 @@ export class RevAiStreamingClient extends EventEmitter {
      * Begins a streaming connection. Returns a duplex
      * from which the user can read responses from the api and to which the user
      * should write their audio data
+     * @param metadata (optional) Metadata to be associated with the streaming job
      *
      * @returns BufferedDuplex. Data written to this buffer will be sent to the api
      * Data received from the api can be read from this duplex
      */
-    public start(): Duplex {
+    public start(metadata?: string): Duplex {
```

Comment:
For extensibility, we may want to make this parameter an object with a metadata property.

The object could be called SessionConfig.

This way as we add more configuration settings the ordering of parameters won't matter.